app/Http/Controllers/TransactionController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use Illuminate\Http\Request;
+use App\Models\transaction;
+use PDF;
+
+class TransactionController extends Controller
+{
+    public function index(){
+        return view('dashboard.createTransaction');
```

Comment:
```suggestion
        return view('dashboard.create_transaction');
```
Use snake_case to name your blade files,
Ref: https://stackoverflow.com/a/61811356

Reply:
Make similar changes throughout

Reply:
Thanks for the feedback sir, I'll update code in the latest commit.